src/program.js

```diff
@@ -76,7 +76,7 @@ export class Program {
     return this;
   }
 
-  async run(absolutePackageDir: string,
+  async execWebExt(absolutePackageDir: string,
```

Comment:
In this case the web-ext part is implied. I think it would make sense to call this `execute` because it gets called like `program.execute()` which makes more sense.